docs/installation.md

```diff
@@ -182,10 +177,12 @@ Get any errors? [See our troubleshooting tips.](#troubleshooting)
   (a way of running software in an isolated environment) and "images" (a
   snapshot of all of the files neccessary to run a container).
 - **Docker Compose**: Compose allows you to configure and run a collection of
-  connected containers (like a web application and it's database)
-- **Docker Machine**: Docker only runs natively on Linux and Windows. On OS X,
-  we'll use Docker Machine to start the Docker server in a virtual linux
-  environment (using Virtualbox)
+  connected containers (like a web application and its database)
+- **Docker Machine**: In environments where Docker Engine is not supported, or
+  where it is not approved for use, Docker Machine can be used to create and
+  manage Docker hosts on virtual machines. For more information on Docker
+  Machine, see the 'How do I use Docker Machine' section in the
+  [usage guide](usage).
```

Comment:
Can we use [the section link](https://cfpb.github.io/cfgov-refresh/usage/#usage-docker) here?